uber/site_sections/signups.py

```diff
@@ -102,11 +103,11 @@ def hotel_requests(self, session, message='', decline=None, **params):
         day_before = (c.EPOCH - timedelta(days=1)).strftime('%A')
         last_day = c.ESCHATON.strftime('%A').upper()
         day_after = (c.ESCHATON + timedelta(days=1)).strftime('%A')
```

Comment:
We previously just grabbed config out of the dictionary of global variables.  Now we do the more sensible ``getattr`` on our single global ``c`` object.